### DISCUSSION TITLE
How to turn PiJuice totally off.

### DISCUSSION THREAD

**Discussion by EnnPeeTee** [PiSupply/PiJuice#1066] [General]
I wonder if anyone can advise me please.

I have assigned the command SYS_FUNC_HALT_POW_OFF  to the long press 1 on SW1 and this indeed seems to do the job of shutting most things down..

However, the brilliant PiJuice still flashes the green LED very briefly every 4 seconds or so and gives a longer full brightness flash once in a while.   Other than removing the battery, is there a method to switch the PiJuice off totally? 

Any guidance will always be gratefully appreciated!

**Comment by tvoverbeek** [comment]
Short answer: No, removing the battery is the only way.
The MCU goes to a low power state (with a stark reduced clock, hence the long flashes),
This is to keep the RTC function in the MCU firmware active and to react on SW presses.

**Reply by EnnPeeTee** [reply]
Brilliant! Thank you for such a speedy reply.

I should explain the reason for my question.  I have produced two identical systems for my boat, with one as a backup which will normally be in store. It will be periodically powered up to update my s/w and of course Pi system s/w.

Thank you again for your very kind assistance,
